de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -105,6 +105,9 @@
     @Inject
     private IWorkspace workspace;
 
+    @Inject
+    private ProjectAPI projectAPI;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "projectAPI" transient or serializable. ([squid:S1948](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1948))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=238&issues=AWYa7GQcvOWp7Yd62c1L&open=AWYa7GQcvOWp7Yd62c1L)

[ISSUE_KEY]: AWYa7GQcvOWp7Yd62c1L